requirements.txt

```diff
@@ -1,2 +1,3 @@
 # generated from manifests external_dependencies
+odoo_test_helper
```

Comment:
see comment in the manifest file

Reply:
The correct way to add this is in a file test-requirements.txt